django/core/management/commands/inspectdb.py

```diff
@@ -150,13 +158,8 @@ def handle_inspection(self, options):
 
                     # Add primary_key and unique, if necessary.
                     if column_name == primary_key_column:
-                        extra_params["primary_key"] = True
-                        if len(primary_key_columns) > 1:
-                            comment_notes.append(
-                                "The composite primary key (%s) found, that is not "
-                                "supported. The first column is selected."
-                                % ", ".join(primary_key_columns)
-                            )
+                        if len(primary_key_columns) == 1:
+                            extra_params["primary_key"] = True
```

Comment:
Do we need `len(primary_key_columns) == 1`? `primary_key_column` is only set if `len(primary_key_columns) == 1`
```suggestion
                        extra_params["primary_key"] = True
```